src/index.html

```diff
@@ -12,6 +12,115 @@
   </head>
   <body>
     <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" 
+    method="post"></form>
+
+    <fieldset class="fieldset">
+            <legend> Personal information </legend>
+            
+            <label class="form_field">
```

Comment:
indentation is too big, 2 tabs is enough

```suggestion
    <fieldset class="fieldset">
      <legend> Personal information </legend>
      <label class="form_field">
```